src/Adapter/ApcCache.php

```diff
@@ -30,36 +30,26 @@ class ApcCache extends BaseApcCache
      */
     protected $token;
 
-    /**
-     * Constructor.
-     *
-     * @param RouterInterface $router  A router instance
-     * @param string          $token   A token to clear the related cache
-     * @param string          $prefix  A prefix to avoid clash between instances
-     * @param array           $servers An array of servers
```

Comment:
could be kept and replaced with `string[]` or whatever the type of the elements is

Reply:
added.